uaclient/messages/__init__.py

```diff
@@ -2554,3 +2558,13 @@ def __repr__(self):
     "error-updating-esm-cache",
     t.gettext("Error updating ESM services cache: {error}"),
 )
+
+E_ENTITLEMENTS_DIRECTIVES_ARE_NOT_UNIQUE = FormattedNamedMessage(
+    "entitlements-definitions-are-not-unique",
```

Comment:
There is a name mismatch here
```suggestion
E_ENTITLEMENTS_DIRECTIVES_ARE_NOT_UNIQUE = FormattedNamedMessage(
    "entitlements-directives-are-not-unique",
```

Reply:
Separately, this message is pretty specific to APT sources and is not applicable to differences in directives in general (e.g. requiredSnaps or additionalPackages), so "entitlement directives are not unique" is a little to generic. Should we rename all of the functions/names in this PR to be more specific to apt sources?

Reply:
Indeed, that's a good point

Reply:
I think the message still needs renamed to match the new exception name
```suggestion
E_ENTITLEMENTS_APT_DIRECTIVES_ARE_NOT_UNIQUE = FormattedNamedMessage(
    "entitlements-apt-directives-are-not-unique",
```